sys/drv/stdvga.c

```diff
@@ -122,47 +122,38 @@ static int stdvga_set_fbinfo(stdvga_state_t *vga, fb_info_t *fb_info) {
   return 0;
 }
 
-static int stdvga_open(vnode_t *v, int mode, file_t *fp) {
-  stdvga_state_t *vga = devfs_node_data(v);
+static int stdvga_open(devnode_t *dev, file_t *fp, int oflags) {
```

Comment:
`stdvga_open` should fail if the user wants to open it for reading since `stdvga_read` is not provided.

Reply:
Done!